### PR TITLE
Add fastlane's SPM integration instructions

### PR DESCRIPTION
**Pull request by minuscorp**
This PR updates the integration with Swift documents to add the SPM support, sorry for the late update 😅

### CODE REVIEW

**Review by rogerluan — APPROVED** [review]
I can't believe we didn't have this before 😂 nice!

File: docs/getting-started/ios/fastlane-swift.md [hunk 1]
```diff
@@ -42,6 +42,59 @@ Run `fastlane <laneName>` in your terminal to execute _fastlane_.
 
 🎉
 
+## Get Started (SPM) (Beta)
+
+Swift Package Manager support is included as a preliminar support which allows the combination of other SPM packages with _fastlane_.
```
Comment (lyndsey-ferguson) [inline docs/getting-started/ios/fastlane-swift.md:47]: This sentence needs clarification.

**Review by lyndsey-ferguson — CHANGES_REQUESTED** [review]

File: docs/getting-started/ios/fastlane-swift.md (hunk 1 above)
Comment (minuscorp) [reply docs/getting-started/ios/fastlane-swift.md:47]: Will do will do, my bad 🙇🏻‍♂️

Comment (rogerluan) [reply docs/getting-started/ios/fastlane-swift.md:47]: I totally thought `preliminar` was an English word 😂 two individuals that speak latin-like languages writing 🇪🇸  + reviewing 🇧🇷 the same code →  💥 😂😂

**Comment by minuscorp** [comment]
@lyndsey-ferguson can you check if it is well written now? 🙏 ❤️

File: docs/getting-started/ios/fastlane-swift.md [hunk 2]
```diff
@@ -42,6 +42,59 @@ Run `fastlane <laneName>` in your terminal to execute _fastlane_.
 
 🎉
 
+## Get Started (SPM) (Beta)
+
+Swift Package Manager support is included as a preliminary feature that allows the usage of _fastlane_ with other SPM packages.
```
Comment (lyndsey-ferguson) [inline docs/getting-started/ios/fastlane-swift.md:47]:
```suggestion
Swift Package Manager support is included as a preliminary feature to integrate _fastlane_ with other SPM packages.
```

Comment (lyndsey-ferguson) [reply docs/getting-started/ios/fastlane-swift.md:47]: @minuscorp this _seems_ better to me, but does it accurately depict what you're suggesting?

Comment (minuscorp) [reply docs/getting-started/ios/fastlane-swift.md:47]: _fastlane_ is distributed as a package that allows third-party packages to introduce its usage through SPM. Am I explaining well? 😟

Comment (lyndsey-ferguson) [reply docs/getting-started/ios/fastlane-swift.md:47]: This seems like a complicated concept to get across, and my mind is _relatively_ fresh this morning, but I'm lost. Perdón, pero no lo entiendo, a lo mejor es más facil explicar en español primero, y despues en ingles? Me preocupa que alguien, como yo, no lo va a entender.

Comment (minuscorp) [reply docs/getting-started/ios/fastlane-swift.md:47]: El paquete _fastlane_ se distribuye como un paquete SPM, de tal forma que otros paquetes puedan crear ejecutables que lo integren. 
Así queda algo más claro? Es que en una línea es un poco complicado por cómo es la integración de SPM en si 😪

File: docs/getting-started/ios/fastlane-swift.md [hunk 3]
```diff
@@ -42,6 +42,59 @@ Run `fastlane <laneName>` in your terminal to execute _fastlane_.
...
+### Step 1
+
+Create an executable Swift Package Manager project with `swift package init --type executable`.
+
+### Step 2
+
+Add the _fastlane_ dependency to your `Package.swift`.
+
+```swift
+.package(name: "Fastlane", url: "https://github.com/fastlane/fastlane", .from("2.179.0"))
```
Comment (TimOliver) [inline docs/getting-started/ios/fastlane-swift.md:58]: Ahh, I just tried to add this line to my own `Package.swift` file and it said it was invalid. Here's what I think the correct syntax needs to be.

```suggestion
.package(name: "Fastlane", url: "https://github.com/fastlane/fastlane", from: "2.179.0")
```

Hope that helps! :)

File: docs/getting-started/ios/fastlane-swift.md [hunk 4]
```diff
@@ -42,6 +42,59 @@ Run `fastlane <laneName>` in your terminal to execute _fastlane_.
...
+
+```swift
+import Fastlane
+
+Main().run(with: Fastfile())
+```
+
+### Step 5
+
+Modify the target of your executable to have executable arguments `lane myLane` or add them in the call after making `swift build`.
```
Comment (TimOliver) [inline docs/getting-started/ios/fastlane-swift.md:86]: With me still being rather super new to all of this, is there a tutorial elsewhere that might explain how to do this? 😅

File: docs/getting-started/ios/fastlane-swift.md (hunk 3 above)
Comment (minuscorp) [reply docs/getting-started/ios/fastlane-swift.md:58]: Yep, I meant `.upToNextMajor(from: "2.179.0")` but it really does not matter, what a day I had🤯

File: docs/getting-started/ios/fastlane-swift.md (hunk 4 above)
Comment (minuscorp) [reply docs/getting-started/ios/fastlane-swift.md:86]: If you generate an Xcode project, you can change the arguments being passed into the executable in the target options or just add them into the generated executable as if it was other CLI.

File: docs/getting-started/ios/fastlane-swift.md [hunk 5]
```diff
@@ -42,6 +42,84 @@ Run `fastlane <laneName>` in your terminal to execute _fastlane_.
...
+```
+
+A full example of a working package description would be the following.
+
+```swift
+// swift-tools-version:5.2
+
+import PackageDescription
+
+let package = Package(
```
Comment (minuscorp) [inline docs/getting-started/ios/fastlane-swift.md:68]: @TimOliver I have updated the step 2 to give a full working example of a package description, see if this helps a bit ❤️

File: docs/getting-started/ios/fastlane-swift.md (hunk 2 above)
Comment (lyndsey-ferguson) [reply docs/getting-started/ios/fastlane-swift.md:47]: ¡Gracias!
```suggestion
Swift Package Manager (SPM) support adds the ability to distribute _fastlane_ as a Swift Package Manager package, which allows other packages to integrate with the toolset.
```
Does that match up with what you're saying?

**Review by lyndsey-ferguson — APPROVED** [review]

**Review by joshdholtz — APPROVED** [review]
🔥🔥🔥🔥🔥

**Pull request merged**